test/server/cards/01_SOR/DarthVaderDarkLordOfTheSith.spec.js

```diff
@@ -0,0 +1,94 @@
+describe('Darth Vader, Dark Lord of the Sith', function() {
+    integration(function() {
+        describe('Vader\'s undeployed ability', function() {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['tieln-fighter', 'swoop-racer'],
+                        groundArena: ['atst'],
+                        leader: 'darth-vader#dark-lord-of-the-sith',
+                        resources: 6 //making vader undeployable makes testing the activated ability's condition smoother
+                    },
+                    player2: {
+                        spaceArena: ['alliance-xwing'],
+                        leader: { card: 'luke-skywalker#faithful-friend', deployed: true }
+                    }
+                });
+            });
+
+            it('should only have an effect if the controller played a villainy card this phase, but still be usable otherwise', function () {
+                //no card played
+                let exhaustedResourcesBeforeAbilityUsed = this.player1.countExhaustedResources();
+                this.player1.clickCard(this.darthVader);
+
+                expect(this.darthVader.exhausted).toBe(true);
+                expect(this.player1.countExhaustedResources()).toBe(exhaustedResourcesBeforeAbilityUsed + 1);
+                expect(this.atst.damage).toBe(0);
+                expect(this.lukeSkywalker.damage).toBe(0);
+                expect(this.allianceXwing.damage).toBe(0);
+                expect(this.p1Base.damage).toBe(0);
+                expect(this.p2Base.damage).toBe(0);
+
+                this.moveToNextActionPhase();
+
+                //play a villainy card
+                this.player1.clickCard(this.tielnFighter);
+                this.player2.pass();
+
+                exhaustedResourcesBeforeAbilityUsed = this.player1.countExhaustedResources();
+                this.player1.clickCard(this.darthVader);
+
+                expect(this.player1).toBeAbleToSelectExactly([this.tielnFighter, this.atst, this.lukeSkywalker, this.allianceXwing]);
+                this.player1.clickCard(this.lukeSkywalker);
+
+                expect(this.player1).toBeAbleToSelectExactly([this.p1Base, this.p2Base]);
+                this.player1.clickCard(this.p2Base);
+
+                expect(this.darthVader.exhausted).toBe(true);
+                expect(this.player1.countExhaustedResources()).toBe(exhaustedResourcesBeforeAbilityUsed + 1);
+                expect(this.atst.damage).toBe(0);
+                expect(this.lukeSkywalker.damage).toBe(1);
+                expect(this.allianceXwing.damage).toBe(0);
+                expect(this.p1Base.damage).toBe(0);
+                expect(this.p2Base.damage).toBe(1);
+
+                this.moveToNextActionPhase();
+
+                //play a non-villainy card
```

Comment:
I'm okay skipping this test here, I just did it in the Luke tests since that was the first test using the watcher